packages/components/src/unit-control/stories/index.tsx

```diff
@@ -0,0 +1,162 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { UnitControl } from '../';
+import { CSS_UNITS } from '../utils';
+
+const meta: ComponentMeta< typeof UnitControl > = {
+	component: UnitControl,
+	title: 'Components (Experimental)/UnitControl',
+	argTypes: {
+		size: {
+			control: { type: 'select' },
+		},
+		onChange: {
+			control: { type: null },
+		},
+		onUnitChange: {
+			control: { type: null },
+		},
+	},
+	parameters: {
+		controls: {
+			expanded: true,
+			exclude: [ '__unstableInputWidth', '__unstableStateReducer' ],
```

Comment:
@mirka should we hide these unstable props, or should we show them in Storybook?